velox/common/caching/tests/AsyncDataCacheTest.cpp

```diff
@@ -1170,18 +1170,14 @@ TEST_F(AsyncDataCacheTest, shutdown) {
   ASSERT_GT(cache_->ssdCache()->testingTotalLogEvictionFilesSize(), 0);
 
   // Shutdown cache.
-  const auto bytesWrittenBeforeShutdown =
```

Comment:
Can we wait for the pending writes to finish?

Reply:
There is
```
bool writeInProgress() const 
```
Can you provide a test method: testingWaitForWriteToFinish()?
Also can you add one more test just try to shutdown SSD cache right after writing a large amount of data and expect the written bytes after shutdown is no larger than before shutdown? Thanks!